src/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
```

Comment:
- [ ] Kindly Import `createStore` also from `react`.
- **Requirement** Imports `createStore` and `combineReducers `from `redux`.
- [ ] Use `createStore` to create a store through the reducer created in the below step.